mesonbuild/compilers/compilers.py

```diff
@@ -453,18 +453,17 @@ class CompileResult(HoldableObject):
     """The result of Compiler.compiles (and friends)."""
 
     def __init__(self, stdo: T.Optional[str] = None, stde: T.Optional[str] = None,
-                 args: T.Optional[T.List[str]] = None,
+                 command: T.Optional[T.List[str]] = None,
                  returncode: int = 999, pid: int = -1,
                  text_mode: bool = True,
```

Comment:
text_mode is also unused, fwiw.

Reply:
Yeah, was following up on that. There was a third one that you didn't notice was also not used.